content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -221,6 +221,11 @@ systemctl enable --now kubelet
     
 {{% /tab %}}
 {{% tab name="Container Linux" %}}
+
+  **Notes:**
+
+  - You are supposed to run the following command with privileged user
+
```

Comment:
we recently had discussions that `sudo` should be explicitly added where needed instead of the generic note about priv./root users.

Reply:
If we want to add sudo instead ,there are two commands need to be changed. 
[core@localhost bin]$ sudo curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed "s:/usr/bin:/opt/bin:g" > /etc/systemd/system/kubelet.service
-bash: /etc/systemd/system/kubelet.service: Permission denied
(23) Failed writing body
Here is one possible fix:
sudo curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" -o /etc/systemd/system/kubelet.service&&sudo sed -i "s:/usr/bin:/opt/bin:g" /etc/systemd/system/kubelet.service
Btw,you may notice that the guide for Centos,RHEL,Fedora also should have sudo added.
I'm gonna to update the commit accordingly after it's been decided.

Reply:
adding sudo to the other tabs in this same PR is fine as long as the PR / commit titles are renamed.

Reply:
Best to limit what runs as root, eg:
```shell
curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed -i "s:/usr/bin:/opt/bin:g" | sudo tee /etc/systemd/system/kubelet.service
```